core/admin/templates/admin_general_metabox_contents_espresso_sponsors.template.php

```diff
@@ -1,11 +1,38 @@
 <div class="padding">
-    <p>Upgrade to an <a
+    <p>
+        <?php
+        printf(
+            esc_html__(
+                'Upgrade to an %1$sEvent Espresso 4 support license%2$s to get access to dedicated support staff and premium features, such as a calendar, printable tickets and check-in capabilities, MailChimp integration, and more!',
+                'event_espresso'
+            ),
+            '<a
             href="https://eventespresso.com/pricing/?ee_ver=ee4&utm_source=ee4_plugin_admin&utm_medium=link&utm_campaign=event_espresso_highlights_widget&utm_content=link"
-            target="_blank">Event Espresso 4 support license</a> to get access to dedicated support staff and premium
-        features, such as a calendar, printable tickets and check-in capabilities, Mailchimp integration, and more!</p>
-    <a href="https://eventespresso.com/?ee_ver=ee4&utm_source=ee4_plugin_admin&utm_medium=link&utm_campaign=event_espresso_highlights_widget&utm_content=banner"
-       target="_blank"><img src="<?php echo EE_GLOBAL_ASSETS_URL; ?>images/manage-events-faster-240x240.jpg"/></a>
-    <p>Visit <a
+            target="_blank">',
+            '</a>'
+        );
+        ?>
+    </p>
+    <p>
+        <a href='https://eventespresso.com/?ee_ver=ee4&utm_source=ee4_plugin_admin&utm_medium=link&utm_campaign=event_espresso_highlights_widget&utm_content=banner'
+           target='_blank'>
+            <img alt="<?php esc_html_e('manage events faster', 'event_espresso'); ?>"
+                 src='<?php echo EE_GLOBAL_ASSETS_URL; ?>images/manage-events-faster-240x240.jpg'
+            />
+        </a>
```

Comment:
May be `noreferrer noopener`? 🤔

Reply:
not sure whether they want those added for campaign links